src/main/scala/tech/cryptonomic/nautilus/metering/config/MeteringApiConfig.scala

```diff
@@ -0,0 +1,3 @@
+package tech.cryptonomic.nautilus.metering.config
+
+case class MeteringApiConfig(host: String, port: Int, keys: List[String])
```

Comment:
Can we please add a docstring to this? It is possible that someone confuses `keys` with NC `apiKeys`.

Reply:
done